frontend-v2/src/components/TextField.tsx

```diff
@@ -74,9 +72,7 @@ function TextField<T extends FieldValues>({
               autoShrink !== undefined ? { shrink: autoShrink } : {}
             }
             variant="outlined"
-            value={
-              fieldValue === undefined || fieldValue === null ? "" : fieldValue
-            }
+            value={fieldValue || defaultValue}
```

Comment:
There's a small bug here, in that deleting all the text from the text box will instantly replace it with the default value. Meaning that you can't easily delete the default value, and replace it.